resources/access.rb

```diff
@@ -24,18 +24,18 @@
 property :access_user,   String, required: true, default: 'postgres'
 property :access_addr,   [String, nil], default: nil
 property :access_method, String, required: true, default: 'ident'
-property :notification,  Symbol, required: true, default: :reload
 
 action :grant do
+  config_resource = new_resource
   with_run_context :root do # ~FC037
     edit_resource(:template, "#{conf_dir}/pg_hba.conf") do |new_resource|
       source new_resource.source
       cookbook new_resource.cookbook
       owner 'postgres'
       group 'postgres'
       mode '0600'
-      variables['pg_hba'] ||= {}
-      variables['pg_hba'][new_resource.name] = {
+      variables[:pg_hba] ||= {}
```

Comment:
Why symbols instead?

Reply:
I prefer them because memory usage is better. Minor optimization I know, but I have the opportunity and I wanted to take it. As far as I know, other precedent hasn't been set in the released code for this.